pkg/semtech-loramac/patches/0004-fix-uninitialized-variable.patch

```diff
@@ -0,0 +1,25 @@
+From 9bf864033dfe765f0e06603e273c4665af777a1a Mon Sep 17 00:00:00 2001
+From: Jose Alamos <jose.alamos@haw-hamburg.de>
+Date: Tue, 24 Jul 2018 11:47:59 +0200
+Subject: [PATCH] fix uninitialized variable
+
+---
+ src/mac/LoRaMac.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/src/mac/LoRaMac.c b/src/mac/LoRaMac.c
+index 4cb2c50..46f7661 100644
+--- a/src/mac/LoRaMac.c
++++ b/src/mac/LoRaMac.c
+@@ -3266,6 +3266,8 @@ LoRaMacStatus_t LoRaMacMcpsRequest( McpsReq_t *mcpsRequest )
+             break;
+         }
+         default:
++            /* Shouldn't happen */
```

Comment:
instead of adding a patch file, do you think it's possible to disable the warning for this module in the corresponding package makefile ? This would make maintenance easier when the package version needs to be updated.

Reply:
Then you also need to review all the disabled warnings on update, which is usually not done. It is not more complex than doing a "rebase" so I prefer a patch when it's easy like this.

Also if I am not mistaken, I think @jia200x said it is fixed upstream but that upgrading the package would be more complicated.